scheduled-jobs/build/poll-payload/Jenkinsfile

```diff
@@ -31,6 +31,18 @@ def startPreReleaseJob(String releaseStream, Map latestRelease, Map previousRele
     )
 }
 
+// collect latest rpms and populate the files on mirror
+def publishLatestRpms(String buildVersion) {
```

Comment:
Scheduled jobs are meant to schedule other jobs instead of do the actual work itself. Could you move this part to a separate job?